addon/components/notification-message.js

```diff
@@ -20,7 +20,6 @@ export default Ember.Component.extend({
     // Apply the clear animation duration rule inline
     notificationClearDuration: Ember.computed('notification.clearDuration', function() {
         return Ember.String.fmt("-webkit-animation-duration: %@ms; animation-duration: %@ms",
-          this.get('notification.clearDuration'),
```

Comment:
I think you actually need both of these... Either that or replace both the %@ with %@1http://emberjs.com/api/classes/Ember.String.html#method_fmt